L1Trigger/L1TGlobal/interface/L1TGlobalUtil.h

```diff
@@ -165,7 +167,7 @@ namespace l1t {
     inline const std::string& gtTriggerMenuComment() const { return m_l1GtMenu->getComment(); }
 
     // Prescale Column
-    inline unsigned int prescaleColumn() const { return m_PreScaleColumn; }
+    inline double prescaleColumn() const { return m_PreScaleColumn; }
```

Comment:
if this is the index of the prescale column, rather than the prescale value, it should stay as an `unsigned int` ?

Reply:
correct. reverting the developer's change.